client/executor/wasmi/src/lib.rs

```diff
@@ -18,27 +18,29 @@
 
 //! This crate provides an implementation of `WasmModule` that is baked by wasmi.
 
-use codec::{Decode, Encode};
+use std::{cell::RefCell, rc::Rc, str, sync::Arc};
+
 use log::{debug, error, trace};
+use wasmi::{
+	memory_units::Pages,
+	FuncInstance, ImportsBuilder, MemoryInstance, MemoryRef, Module, ModuleInstance, ModuleRef,
+	RuntimeValue::{self, I32, I64},
+	TableRef,
+};
+
+use codec::{Decode, Encode};
 use sc_executor_common::{
 	error::{Error, MessageWithBacktrace, WasmError},
 	runtime_blob::{DataSegmentsSnapshot, RuntimeBlob},
 	sandbox,
 	util::MemoryTransfer,
 	wasm_runtime::{InvokeMethod, WasmInstance, WasmModule},
 };
-use sp_core::sandbox as sandbox_primitives;
 use sp_runtime_interface::unpack_ptr_and_len;
+use sp_sandbox::env;
```

Comment:
```suggestion
use sp_sandbox::env as sandbox_env;
```

Or something like that. Just `env` is not that expressive.